src/lib/containers/CertificationQuiz.tsx

```diff
@@ -0,0 +1,213 @@
+
+const CertificationQuiz: React.FunctionComponent = () => {
+  const { accessToken } = useSynapseContext()
+  const handleError = useErrorHandler()
+  const [quiz, setQuiz] = useState<Quiz | undefined>()
+  const [questionResponse, setQuestionResponse] = useState<QuestionResponse[]>(
+    [],
+  )
+  const [passingRecord, setPassingRecord] = useState<PassingRecord>()
+  const formRef = React.useRef<HTMLFormElement>(null)
+
+  const GETTING_STARTED_URL =
+    'https://help.synapse.org/docs/Getting-Started.2055471150.html'
+
+  const getQuiz = async () => {
+    try {
+      setQuiz(await SynapseClient.getCertifyQuiz(accessToken))
+    } catch (err: any) {
+      handleError(err)
+    }
+  }
+
+  useEffect(() => {
+    getQuiz()
+  }, [accessToken])
+
+  const onUpdateAnswer = (questionIndex: number, answer: number) => {
+    const newState = [
+      ...questionResponse.filter(obj => obj.questionIndex !== questionIndex),
+      {
+        questionIndex,
+        answerIndex: [answer],
+        concreteType: MULTICHOICE_RESPONSE_CONCRETE_TYPE_VALUE,
+      },
+    ]
+    setQuestionResponse(newState)
+  }
+
+  const handleRetakeQuiz = () => {
+    formRef.current?.reset()
+    setQuestionResponse([])
+    setPassingRecord(undefined)
+    getQuiz()
+  }
+
+  const handleSubmit = async () => {
+    try {
+      if (quiz && quiz.questions.length === questionResponse.length) {
+        const quizResponse: QuizResponse = {
+          quizId: quiz.id,
+          questionResponses: questionResponse,
+        }
+        const passRec = await SynapseClient.postCertifiedUserTestResponse(
+          accessToken,
+          quizResponse,
+        )
+        setPassingRecord(passRec)
+        window.scrollTo(0, 0)
+      } else {
+        displayToast(
+          'Please answer all of the questions and try again.',
+          'warning',
+        )
+      }
+    } catch (err: any) {
+      displayToast(err.reason as string, 'danger')
+    }
+  }
+
+  const actionButtonConfig = (helpUrl: string) =>
+    helpUrl
+      ? {
+          content: <>More info</>,
+          closePopoverOnClick: true,
+          onClick: () => window.open(helpUrl, '_blank'),
+          variant: 'primary',
+        }
+      : undefined
+
+  return (
+    <div className="bootstrap-4-backport CertificationQuiz">
+      {passingRecord && (
+        <div>
+          {!passingRecord.passed && (
+            <div className="failBanner">Quiz Failed</div>
+          )}
+          <Typography variant="hintText">
+            Score: {passingRecord.score} / {quiz?.questions.length}
+          </Typography>
+          {passingRecord.passed ? (
+            displayToast(
+              `You passed the Synapse Certification Quiz on ${passingRecord.passedOn}`,
+              'success',
+            )
+          ) : (
+            <Typography variant="body1">
+              Please review the items shown in red below, and{' '}
+              <a style={{ color: '#407ba0' }} onClick={handleRetakeQuiz}>
```

Comment:
You can get the standardized link styling if you have an href. Then preventDefault so it doesn't do anything
```suggestion
              <a href="#" onClick={e => { 
                e.preventDefault()
                handleRetakeQuiz()
              }}>
```